frontend/Panel.js

```diff
@@ -331,9 +339,7 @@ class Panel extends React.Component {
         extraTabs={extraTabs}
         preferencesPanelShown={this.state.preferencesPanelShown}
         theme={theme}
-        onViewElementSource={
-          this.props.showElementSource ? this.viewElementSource.bind(this) : null
-        }
+        onViewElementSource={this.hasLaunchEditor ? this.viewElementSource.bind(this) : null}
```

Comment:
Let's assume it is set before Panel is rendered for the first time. I *think* that would be the case in CRA environment. So let's just assume that reading it once at startup is safe enough.